src/helpers/constant.js

```diff
@@ -0,0 +1,83 @@
+export default {
```

Comment:
Good initiative but I don't see any much importance of this file, I think the code will be more readable if you just use status code on the controller and displaying error messages in the controller.